src/uncrustify.cpp

```diff
@@ -384,7 +385,7 @@ int main(int argc, char *argv[])
    // If ran without options show the usage info and exit */
    if (argc == 1)
    {
-      usage_exit(nullptr, argv[0], EXIT_SUCCESS);
+      return(usage(nullptr, argv[0], EXIT_SUCCESS));
```

Comment:
I was going to do that originally, but the exit code is used in `usage` other than just to call `exit`. Although, looking more closely, I see now that it is used in a way that somewhat conflates with `argv0`. I originally did it this way because the alternative seemed to be:

```c++
usage(nullptr, argv[0], EXIT_SUCCESS);
return(EXIT_SUCCESS);
```

...but now that I look closer, I think maybe I can avoid this.